org.pymol.PyMOL.yaml

```diff
@@ -7,7 +7,8 @@ finish-args:
   - --share=ipc
   - --socket=x11
   - --device=dri
-
+  - --share=network
+  - --socket=session-bus
```

Comment:
You should actually not allow the session-bus but rather the specific talk names.

Reply:
How do I know what talk name it will be?

Reply:
`--log-session-bus` passed to flatpak run.

Reply:
but I didn't get the dbus error here.

Reply:
I did not get the dbus error too, I will just give it network permission.